packages/cozy-harvest-lib/src/services/utils.js

```diff
@@ -47,6 +47,31 @@ export const fetchAccountsForCipherId = async (cozyClient, cipherId) => {
   return accounts
 }
 
+export const fetchKonnectorFromAccount = async (cozyClient, account) => {
+  const konnectorDoctype = 'io.cozy.konnectors'
+  const slug = account.account_type
+  const konnector = await cozyClient.query(
+    cozyClient.get(konnectorDoctype, slug)
+  )
+  return konnector.data
+    ? {
+        _id: konnector.data._id,
+        _type: konnector.data.type,
+        ...konnector.data.attributes
+      }
+    : null
```

Comment:
Why is the konnector data transformed here ? If you think that would be useful for other applications, it could be useful to have this logic in cozy-client directly (otherwise every other application will transform the data slightly differently, and it's harder to maintain).

Reply:
I agree, but I'm a bit worried that existing code already rely on this format. Maybe @doubleface has insights on this?

Reply:
The io.cozy.konnectors doctype is not used by connectors et connector do not use cozy-client yet

Reply:
I added a issue here https://github.com/cozy/cozy-client/issues/788